File: app/src/main/java/com/prashant/masterbuddy/ws/model/DataPart.java
package com.prashant.masterbuddy.ws.model;

/**
 * Created by prashant.mishra on 22/05/18.
 */

public class DataPart {
    private String fileName;
    private byte[] content;
    private String type;

    public DataPart() {
    }

    public DataPart(String name, byte[] data) {
        fileName = name;
        content = data;
    }

    public String getFileName() {
        return fileName;
    }

    public byte[] getContent() {
        return content;
    }

    public String getType() {
        return type;
    }

}

File: app/src/main/java/com/prashant/masterbuddy/ws/model/DataPart.kt
package com.prashant.masterbuddy.ws.model

/**
 * Created by prashant.mishra on 22/05/18.
 */

class DataPart(name: String, data: ByteArray) {
    val fileName: String = name
    val content: ByteArray = data
    val type: String? = null

}
